content/cran-mirror/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Mirror of CRAN packages"
+date: 2018-01-28T22:01:36+01:00
+anchor: "cranatgh"
+weight: 500
+---
+
+R-hub unofficially mirrors CRAN packages to the [`cran` GitHub organization](https://github.com/cran/). Each CRAN package lives in its own GitHub repository, e.g. package Rcpp is at https://github.com/cran/Rcpp. All versions of all packages are included, and each version is a separate git commit, tagged with the version number. We tried to keep the original dates and authors/maintainers as well.
+
+The main goal of CRAN @ GitHub is the **easy access to the source code or CRAN packages**. In particular, you can
+
+* See differences between package versions, e.g. [what changed between 1.2.0 and 1.2.1](https://github.com/cran/crayon/commit/72b56a09cd4cbc787eb90a31d88f736d280af8b5) in crayon.
+
+* Fork packages, and create patched versions of them, for your own purposes. 
+
+* Use [GitHub search functionality](https://help.github.com/en/articles/about-searching-on-github).
+
+To find out how things work under the hood, [refer to `cranatgh` GitHub repository](https://github.com/metacran/cranatgh/). R-hub mirror of CRAN packages is one of the sources used by [Jim Hester's `lookup` package](https://github.com/jimhester/lookup), that helps you lookup R full function definitions, including compiled code, S3 and S4 methods. 
```

Comment:
I think we don't need to mention cranatgh at all.

Reply:
Ok so I'd just mention it in the blog post about the "mirror"

Reply:
maybe, or maybe not even there.....

Reply:
You can mention lookup, that's sg people can actually use. cranatgh is useless for people, unless they want to create their own GH mirror....

Reply:
I am pretty sure a few people would be interested in the technical details 😉

Reply:
Ok, in the blog post then.

In general I am fine with omitting details that few people would use, from the docs. I am also worried that the docs go out of date...